views/error/service-offline.html

```diff
@@ -0,0 +1,19 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Service offline - File a confirmation statement
```

Comment:
I see that we are using this offline page using an example from confirmation statement. Could you please customize it with the text relevant for our new test micro-service.

Reply:
Ah, I missed this reference. I'll Change it now and make another commit.